packages/react-generator/src/generator/page.js

```
@@ -0,0 +1,761 @@
+      return attrsArr.push(`${key}={state.${valueKey}}`)
+    }
+    
+    const parsedValue = unwrapExpression(JSON.stringify(item))
+    return attrsArr.push(`${key}={${parsedValue}}`)
+  }
+
+  return attrsArr.push(`${key}={${item}}`)
+}
+
+// 工具函数：处理数据绑定
+const handleJSXDataBinding = ({ key, item, attrsArr, description, state, isClassComponent = false, hasOnChange = false, componentName = '' }) => {
```

ESLint

[error] 'description' is defined but never used. Allowed unused args must match /^_/u.

(@typescript-eslint/no-unused-vars)